dkc/core/rest/user.py

```diff
@@ -9,3 +13,17 @@ class Meta:
             'id',
             'username',
         ]
+
+
+class UserViewSet(GenericViewSet):
+    queryset = User.objects.all()
+
+    permission_classes = [AllowAny]
+
+    @action(detail=False, pagination_class=None)
+    def me(self, request):
+        """Return the currently logged in user's information."""
+        if request.user.is_anonymous:
+            return Response(status=204)
```

Comment:
IMO, `404` is more appropriate, but I think we should defer that bikeshedding to #73.